core/src/main/scala/com.snowplowanalytics.stream/loader/Config.scala

```diff
@@ -176,7 +176,7 @@ object Config {
     implicit val esAWSConfigReader: ConfigReader[ESAWSConfig] = deriveReader[ESAWSConfig]
   }
 
-  case class ESClusterConfig(name: String, index: String, documentType: String)
+  case class ESClusterConfig(name: String, index: String, documentType: Option[String])
```

Comment:
This config change needs documentation in the example hocon file.  I know you are improving the hocon in a separate commit, so it is fine if you make that change later, not now.